test/integration/suite1/fixtures/many-to-many-extended-cases.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
base.json equals this file except of the relations or?

each suite has a base. and then a test can insert more data.

Reply:
oh btw: let me know if you think the concept does not make sense to you :)

Reply:
yeah, it makes sense but only in M:M relations for now. base.json fills out data for 1:M scenario atm, so `one-to-many-extended-cases.json` - doesn't have any use case at the moment (maybe there will be needed more data and then that file could be used to kind of extend the base with more records).